### PR TITLE
catgirl toxin

### PR DESCRIPTION
**Pull request by actioninja**
Read the title.  Exactly what it sounds like, has been tested and shouldn't cause issues because it's just an extension of the existing mutation toxins.  I think this also adds it to the strange seed pool but I couldn't exactly figure out how that works so I'm not sure.

:cl: actioninja
add: Added (unobtainable) Felinid Mutation Toxin.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -457,6 +457,13 @@
 	race = /datum/species/jelly/slime
 	mutationtext = "<span class='danger'>The pain subsides. Your whole body feels like slime.</span>"
 
+/datum/reagent/mutationtoxin/felinid
+	name = "Felinid Mutation Toxin"
+	id = "felenidmutationtoxin"
+	color = "#5EFF3B" //RGB: 94, 255, 59
+	race = /datum/species/human/felinid
+	mutationtext = "<span class='danger'>The pain subsides. You feel... like a degenerate.</span>"
```
Comment (Borisvanmemes) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Now THAT, is funny.

**Comment by XDTM** [comment]
The strange seed pool pulls from all chemicals where blacklisted is not TRUE, so this does in fact add it to the random pool

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -457,6 +457,13 @@
 	race = /datum/species/jelly/slime
 	mutationtext = "<span class='danger'>The pain subsides. Your whole body feels like slime.</span>"
 
+/datum/reagent/mutationtoxin/felinid
+	name = "Felinid Mutation Toxin"
+	id = "felenidmutationtoxin"
```
Comment (XDTM) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: felinid

Comment (actioninja) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: That's what happens when you make shitpost additions at 3am I guess.

**Comment by Iamgoofball** [comment]
@XDTM the species changers ARE blacklisted

**Comment by actioninja** [comment]
I rolled skeleton toxin like a week ago, is it specific ones that are blacklisted?

**Comment by XDTM** [comment]
@Iamgoofball they are not

**Comment by Iamgoofball** [comment]
what the fuck

they probably should be blacklisted, thats weird

**Comment by XDTM** [comment]
@Iamgoofball while i kinda agree, i think nobody fixed it because it can lead to fun situations and is not easily beaten into the ground every round due to its rarity.

**Comment by ExcessiveUseOfCobblestone** [comment]
Can you explain the point of the feature in the PR desc

**Comment by XDTM** [comment]
Probably not even worth a changelog considering it's not regularly obtainable

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -457,6 +457,13 @@
 	race = /datum/species/jelly/slime
 	mutationtext = "<span class='danger'>The pain subsides. Your whole body feels like slime.</span>"
 
+/datum/reagent/mutationtoxin/felinid
+	name = "Felinid Mutation Toxin"
+	id = "felinidmutationtoxin"
+	color = "#5EFF3B" //RGB: 94, 255, 59
+	race = /datum/species/human/felinid
+	mutationtext = "<span class='danger'>The pain subsides. You feel... like a degenerate.</span>"
```
Comment (Militaires) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Ok, this is epic.

**Comment by davethwave** [comment]
Honestly I think the other mutation toxins should be readded somehow to chemistry or something.

**Comment by ExcessiveUseOfCobblestone** [comment]
Even with xenobio which is based on RNG it was too strong, why do you think giving a reliable recipe will make it better?

**Comment by davethwave** [comment]
Besides being annoying the mutation toxin is not that op. In my time of 600 or so hours in tg I've never even seen a chemist mix up android mutation tox for instance. Which leaves more of the rp mutation toxins like lizard people and the like.

**Comment by ExcessiveUseOfCobblestone** [comment]
When was android toxin available to chemists?

**Comment by davethwave** [comment]
Dunno how far back it was added but you used to be able to combine black slime toxin with teslium and one other ingredient to make android mutation toxin.

**Comment by ExcessiveUseOfCobblestone** [comment]
Ofc chemists wouldn't do that, that's xenobio's job which I remember everyone running around as skeleton's more than enough times.

**Comment by XDTM** [comment]
The reason for their removal is that it was jarring to have an arbitrary selection of species with little in common between them, with stuff like androids and skeletons, all available from a single department. I'm all for adding methods to access unique species, but these methods should be as unique as their species.

**Comment by Wilchenx** [comment]
@XDTM how about cat ears and tail in limb grower, we already have stuff specific for other races in that

**Comment by XDTM** [comment]
That's a pretty good place to put them in

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 3 above)
[the remark this answers is not in the document]
Comment (GuyonBroadway) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: I'd hardly call a seldom used admin secret tg culture.

**Comment by fludd12** [comment]
Could make it work like how full augmentation changes you to android

**Comment by XDTM** [comment]
Full aug does not change you into an android, which is itself an issue

**Review by SpaceManiac — APPROVED** [review]
does what it says on the tin

**Pull request merged**